src/Migrations/MigrationOptions.cs

```diff
@@ -46,5 +58,31 @@ public void AddAssemblyScriptsProvider(Assembly assembly, string resourceNamespa
         /// <param name="folderPath">Path to folder where migration scripts are stored.</param>
         public void AddFileScriptsProvider(string folderPath)
             => AddScriptsProvider(new FileMigrationScriptsProvider(folderPath));
+
+        /// <summary>
+        /// Add action to be executed on database after migration scripts are executed.
+        /// </summary>
+        /// <param name="actionToExecute"></param>
+        public void AddAfterMigrationAction(Func<IDatabase, long, Task> actionToExecute)
+        {
+            _actions.Add(actionToExecute);
+        }
+
+        /// <summary>
+        /// Add action of refreshing all database views.
+        /// </summary>
+        /// <param name="scriptName">Name of the script containing query for refreshing all views. Default one is Kros.KORM\Resources\RefreshViews.sql </param>
+        public void AddRefreshViewsAction(string scriptName = DefaultRefreshViewsScriptName)
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            var resourceName = $"{assembly.GetName().Name}.{DefaultResourceNamespace}.{scriptName}";
+            AddAfterMigrationAction(async (database, _) =>
+            {
+                Stream resourceStream = assembly.GetManifestResourceStream(resourceName);
```

Comment:
Nejdem to teraz overovať, ale mám pocit, že `GetManifestResourceStream` vracia nullabe stream, čiže by si mala použiť `Stream?`. Ak je to tak tak prosím ešte over či je to null a ak áno tak vyhoď `InvalidOperationException`.

A rovnako použi prosím `using Stream? ...`. Pretože aj `Stream` je disposable.

Reply:
pri jednom aj druhom usingu podľa mňa môžeš použiť `await using` miesto `using`. Ale istý som si není, neskúšal som to. Prosím skús.